py/selenium/webdriver/edge/webdriver.py

```diff
@@ -14,79 +14,51 @@
-           capabilities only, such as "proxy" or "loggingPref".
-         - port - port you would like the service to run, if left as 0, a free port will be found.
-         - verbose - whether to set verbose logging in the service
-         - service_log_path - Where to log information from the driver.
+         - executable_path - Deprecated: path to the executable. If the default is used it assumes the executable is in the $PATH
+         - capabilities - Dictionary object with non-browser specific capabilities only, such as "proxy" or "loggingPref".
+           Only available in Legacy mode
+         - port - Deprecated: port you would like the service to run, if left as 0, a free port will be found.
+         - verbose - whether to set verbose logging in the service. Only available in Legacy Mode
+         - service_log_path - Deprecated: Where to log information from the driver.
          - keep_alive - Whether to configure EdgeRemoteConnection to use HTTP keep-alive.
+         - service_args - Deprecated: List of args to pass to the driver service
+         - is_legacy: Whether to use MicrosoftWebDriver.exe (legacy) or MSEdgeDriver.exe (chromium-based). Defaults to True.
          """
-        if port != DEFAULT_PORT:
-            warnings.warn('port has been deprecated, please pass in a Service object',
-                          DeprecationWarning, stacklevel=2)
-        self.port = port
-
-        if service_log_path != DEFAULT_SERVICE_LOG_PATH:
-            warnings.warn('service_log_path has been deprecated, please pass in a Service object',
-                          DeprecationWarning, stacklevel=2)
-        if capabilities is not None:
-            warnings.warn('capabilities has been deprecated, please pass in a Service object',
-                          DeprecationWarning, stacklevel=2)
-        if service_log_path != DEFAULT_SERVICE_LOG_PATH:
-            warnings.warn('service_log_path has been deprecated, please pass in a Service object',
-                          DeprecationWarning, stacklevel=2)
-        if verbose:
-            warnings.warn('verbose has been deprecated, please pass in a Service object',
-                          DeprecationWarning, stacklevel=2)
-
-        if service:
-            self.service = service
-        else:
-            self.service = Service(executable_path, port=self.port, verbose=verbose,
-                                   log_path=service_log_path)
-        self.service.start()
-
-        if capabilities is None:
-            capabilities = DesiredCapabilities.EDGE
-
-        RemoteWebDriver.__init__(
-            self,
-            command_executor=RemoteConnection(self.service.service_url,
-                                              resolve_ip=False,
-                                              keep_alive=keep_alive),
-            desired_capabilities=capabilities)
-        self._is_remote = False
+        if not is_legacy:
+            executable_path = "MSEdgeDriver.exe"
```

Comment:
this should be `msedgedriver` as that's the default download case

Reply:
Can you confirm this is right?

Reply:
yes 👍